friction.py

```diff
@@ -127,9 +127,81 @@ def draw(self, context):
         if self.plane_hinge:
             row.prop(self, "radius")
         self.draw_link(layout, "function_name", "function_edit")
+    def check(self, context):
```

Comment:
Thanks for adding def check()

See the comments below for a weblink to the MBDyn input manual. This friction entity has changed, and no longer has a "plane_hinge". We need to cross check against the manual to assure the correct form for each entity as we add them.